lib/auth/auth_with_roles_test.go

```diff
@@ -8398,3 +8398,185 @@ func TestIsMFARequired_AdminAction(t *testing.T) {
 		})
 	}
 }
+
+func TestCloudDefaultPasswordless(t *testing.T) {
```

Comment:
Did you check for a test for the old `len(users)==1` logic, just in case?

Reply:
Yeah, I actually found out that the test servers don't create the preset users, so I created them in the tests to simulate the runtime behavior. Then I added one test that catches the original bug (first user when presets exist) to make sure we're solving it.

TY!